helpers/copy-template-files.js

```diff
@@ -0,0 +1,29 @@
+const cpy = require('cpy')
```

Comment:
Generally if we do significant refactoring like this, it should be a separate change/PR from feature enhancements. That makes it easier to test, review, revert, and/or mitigate issues.

Reply:
@amclin would you like a separate PR for this? If so, could we make sure we get it through the pipeline quickly. I don't mind doing a separate PR.

Reply:
@amclin  see https://github.com/amclin/react-project-boilerplate/pull/160/files